charts/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $namespace := .Values.common.namespace | required "A valid .common.namespace is required!" -}}
+{{- $yunikornHost := .Values.yunikorn.host | required "A valid .yunikorn.host is required!" -}}
+{{- $yunikornPort := .Values.yunikorn.port | required "A valid .yunikorn.port is required!" -}}
+{{- $yunikornProtocol := .Values.yunikorn.protocol | required "A valid .yunikorn.protocol is required!" -}}
+{{- $dbHost := .Values.db.host | required "A valid .db.host is required!" -}}
+{{- $dbPort := .Values.db.port | required "A valid .db.port is required!" -}}
+{{- $dbName := .Values.db.name | required "A valid .db.name is required!" -}}
+{{- $dbUser := .Values.db.user | required "A valid .db.user is required!" -}}
+{{- $dbPoolMaxConns := .Values.db.poolMaxConns | required "A valid .db.poolMaxConns is required!" -}}
+{{- $dbPoolMinConns := .Values.db.poolMinConns | required "A valid .db.poolMinConns is required!" -}}
+{{- $dbPoolMaxConnLifetime := .Values.db.poolMaxConnLifetime | required "A valid .db.poolMaxConnLifetime is required!" -}}
+{{- $dbPoolMaxConnIdleTime := .Values.db.poolMaxConnIdleTime | required "A valid .db.poolMaxConnIdleTime is required!" -}}
+{{- $yhsServerAddr := .Values.yhsServerAddr | required "A valid .yhsServerAddr is required!" -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: yhs-config
```

Comment:
Take a look at this `_helpers.tpl` file - https://github.com/armadaproject/armada-operator/blob/main/charts/armada-operator/templates/_helpers.tpl

Implement similar things for well-defined labels and name.

Take a look at this resource for example:
- https://github.com/armadaproject/armada-operator/blob/main/charts/armada-operator/templates/deployment.yaml#L10
- https://github.com/armadaproject/armada-operator/blob/main/charts/armada-operator/templates/webhook-service.yaml#L14
- https://github.com/armadaproject/armada-operator/blob/main/charts/armada-operator/templates/webhook-service.yaml#L4

Reply:
Implemented similar `_helpers.tpl` to manage the labels